ddl/partition.go

```diff
@@ -1628,6 +1655,29 @@ func checkExchangePartitionRecordValidation(w *worker, pt *model.TableInfo, inde
 	return nil
 }
 
+func checkExchangePartitionPlacementPolicy(t *meta.Meta, ntPlacementPolicyRef *model.PolicyRefInfo, ptPlacementPolicyRef *model.PolicyRefInfo) error {
```

Comment:
(Questions only)
I forgot what we decided around EXCHANGE and placement policies, do you have a link to where it specifies how to deal with it?
I would be OK to allow differences between them, but can also see that it then might need to update the table metadata I assume?

Reply:
We decided to swap when the placement policies were exactly equal, in the documentation of the test plan, in the comments of the test plan documentation.